design_docs/2023_07_20_dynamic_shader_parameter_amount.md

```diff
@@ -0,0 +1,55 @@
+# Passing variables to the shader
+We need to be able to pass an arbitrary number of textures and parameters to the custom shaders. For parameters, we need to be able to pass some "header parameters" -- constants used in all videos, as well as some per-video data.
```

Comment:
I don't like the phrase `"header parameters"`. First what comes to my mind is the request header, which has nothing to do with `constants used in all videos`.

Maybe we should use sth like:
- `transformation params`
- `video params`